models/sibcasa/inst/namel_sibdrv

```diff
@@ -0,0 +1,96 @@
+$INLIST_SIBDRV
+     nsib           = 59167,
+     ztemp          = 100.0,
+     zwind          = 100.0,
+/
+
+$IOLIST_SIBDRV
+      param_path    = @PARAM_FILE_PATH@
+      ic_path       = @INITIAL_CONDITIONS_PATH@
+      dr_format     = "(""@SITE_MET@"",i4.4,i2.2,"".nc"")"
+      out_path      = @OUTDIR@
+      qp_path       = "/projectnb/dietzelab/tonygard/sibcasa/PEcAn/SiB/sibcasa_arctic4/sib_qpopts"
```

Comment:
Are these hard coded paths expected? Will this work on all pecan systems?

Reply:
I'll fix this one when I finish with handling parameters.